model-optimizer/requirements_tf.txt

```diff
@@ -1,6 +1,6 @@
 tensorflow>=1.15.2,<2.0; python_version < "3.8"
 tensorflow>=2.0; python_version >= "3.8"
 networkx>=1.11
-numpy>=1.13.0
+numpy>=1.13.0,<1.19.0
```

Comment:
what about to update other requirement files? requirements_caffe.txt, etc.?

Reply:
This is an issue with TF 1.15.X compatibility. It is not related to other frameworks.

Reply:
OK, I see. It is better to give explicit description what was wrong.